conf/CatalogoDatiInformativi.xsd

```diff
@@ -434,7 +434,7 @@
 					</xs:documentation>
         </xs:annotation>
       </xs:element>
-      <xs:element type="stZeroUno" name="stornoPagamento" fixed="0">
```

Comment:
lo stornoPagamento è una funzionalità non più attiva. Corretto non passare il CDI

Reply:
@furlaniilaria

Reply:
C'è un PSP in produzione che ha lo storno configurato a "true". Quel CDI verrebbe escluso dalla reponse alla nodoChiediInformativaPSP. Perché la modifica non escluda alcun PSP attualmente attivo in Prod, è necessario un CDI con "storno=N" per il PSP in questione.